corebehrt/functional/creators.py

```diff
@@ -52,7 +52,7 @@ def create_death(patients_info: dd.DataFrame) -> dd.DataFrame:
     death_events = death_events.rename(columns={"DEATHDATE": "TIMESTAMP"})
     death_events = death_events.assign(
         concept="Death",
-        ADMISSION_ID=-1,  # Segment for death event
+        ADMISSION_ID="last",  # Segment for death event
```

Comment:
It's just a unique placeholder for the factorize function of segment_creation. Can be anything really, but it's better to just initialize it as a str instead of int and later convert to str.